.github/workflows/autobuild.yml

```
@@ -14,178 +15,169 @@
+        id:                         jamulus-build-vars
+
+      - name:                       Remove release, if existing (for branches)
+        if:                         contains(steps.jamulus-build-vars.outputs.PUBLISH_TO_RELEASE, 'true')
+        continue-on-error:          true
+        uses:                       dev-drprasad/delete-tag-and-release@v0.1.2
+        with:
+          delete_release:           true
+          tag_name:                 ${{ steps.jamulus-build-vars.outputs.RELEASE_TAG }}
+        env:
+          GITHUB_TOKEN:             ${{ secrets.GITHUB_TOKEN }}
 
-     steps:
-         # Checkout code
-         - name:                    Checkout code
-           uses:                    actions/checkout@v2
-
-         # Set variables
-         # Determine release / pre-release
-         - name:                    Get Jamulus build info, determine actions & variables
-           run:                     python3 ${{ github.workspace }}/.github/actions_scripts/analyse_git_reference.py
-           id:                      jamulus-build-vars
-
-         # remove release, if it exists (with this releasetag)
-         - name:                    Remove release, if existing (for branches)
-           if:                      ${{ contains(steps.jamulus-build-vars.outputs.PUBLISH_TO_RELEASE, 'true') }}
-           continue-on-error:       true
-           uses:                    dev-drprasad/delete-tag-and-release@v0.1.2
-           with:
-             delete_release:        true
-             tag_name:              ${{ steps.jamulus-build-vars.outputs.RELEASE_TAG }}
-           env:
-             GITHUB_TOKEN:          ${{ secrets.GITHUB_TOKEN }}
-         # create release (empty, filled by next jobs)
-         - name:                    'Create Release ${{steps.jamulus-build-vars.outputs.RELEASE_TAG}}  {{steps.jamulus-build-vars.outputs.RELEASE_TITLE}}'
-           if:                      ${{ contains(steps.jamulus-build-vars.outputs.PUBLISH_TO_RELEASE, 'true') }}
-           id:                      create_release_step
-           uses:                    actions/create-release@v1
-           env:
-             GITHUB_TOKEN:          ${{ secrets.GITHUB_TOKEN }}
-           with:
-             tag_name:              ${{ steps.jamulus-build-vars.outputs.RELEASE_TAG }}
-             release_name:          ${{ steps.jamulus-build-vars.outputs.RELEASE_TITLE }}
-             body_path:             ${{ github.workspace }}/autoLatestChangelog.md
-             prerelease:            ${{ steps.jamulus-build-vars.outputs.IS_PRERELEASE }}
-             draft:                 false
-
-
-         ### CANCEL ### can be used for development concerning release-creation
-         #- name:                    Cancelthrougherroe
-         #  run:                     myundefinedfunction
+      - name:                       'Create Release ${{steps.jamulus-build-vars.outputs.RELEASE_TAG}}  {{steps.jamulus-build-vars.outputs.RELEASE_TITLE}}'
```

I think a $ is missing:

```suggestion
      - name:                       'Create Release ${{steps.jamulus-build-vars.outputs.RELEASE_TAG}}  ${{steps.jamulus-build-vars.outputs.RELEASE_TITLE}}'
```

---

Great. Thanks.